dcm4che-assembly/pom.xml

```diff
@@ -42,7 +42,7 @@
   <parent>
     <artifactId>dcm4che-parent</artifactId>
     <groupId>org.dcm4che</groupId>
-    <version>3.3.7.8-SNAPSHOT</version>
```

Comment:
It's fine to leave this at 3.3.7.8-SNAPSHOT for now as the previous build was 3.3.7.7 -- after review we can drop "-SNAPSHOT" and build, then bump to 3.3.7.9-SNAPSHOT. (ditto for all other pom's)